### DISCUSSION TITLE
Discussion about UI Component Guide Document

### DISCUSSION THREAD

**Discussion by zion830** [euphony-io/euphony#232] [General]
Now I'm thinking about a detail way how to implement & documentation of `EuphonyTxPanel`. 
Before that, I'd like to debate about **the access path** of the guide document.

My plan is as below.

1. Simple Introdution : Add screenshots of them to the step '[Euphony is very easy to use](https://github.com/euphony-io/euphony/blob/master/README.md#euphony-is-very-easy-to-use)' in README
2. Mode Detail Guide : Add a new markdown file named like **"How to use Basic Ui Components of Euphony"**
    * And add this file's link to README as a new detail guide
<img width="626" alt="스크린샷 2022-09-07 오후 7 48 18" src="https://user-images.githubusercontent.com/35446215/188860270-6eabd765-e046-49b2-b883-b5b4928b1b47.png">

I want your opinions or other good ideas!

**Comment by yeonns** [comment]
@zion830 I think your documentation plan is so nice! How about your implementation plan? I wanna know more 😄 (e.g, component design and interaction, parameter of compose component...)

**Comment by designe** [comment]
@zion830 
That's great idea :) 
I agree with your idea. split simple intro & detail guideline for UI component!

simple is more difficult to express easilly. so, how about to express detail guideline first?
after that, making compression of detail guideline will be simple intro :)
